pyannote/audio/core/task.py

```diff
@@ -172,8 +174,11 @@ def __init__(
         num_workers: int = None,
         pin_memory: bool = False,
         augmentation: BaseWaveformTransform = None,
+        gpu_transforms: bool = True,
     ):
         super().__init__()
+        # gpu transforms
+        self.gpu_transforms = gpu_transforms and torch.cuda.is_available()
```

Comment:
As a follow-up of the above comment (line 177), we no longer have to check whether `cuda` is available.